### PR TITLE
[#181, #180] A few frontend build fixes

### PR DESCRIPTION
**Pull request by lewiscollard**
* Update stylelint-disable lines to use `selector-max-type` and `selector-max-universal` rather than `selector-no-*`.
* Remove a reference to a dead Vue component.
* Force prettier to use single quotes on CSS - it was converting single quotes to double quotes which was causing pre-commits to fail.

### CODE REVIEW

File: {{cookiecutter.repo_name}}/package.json [hunk 1]
```diff
@@ -106,7 +106,7 @@
         "git add"
       ],
       "**/!(definitions)/*.css": [
-        "prettier --write",
+        "prettier --write --single-quote",
```
Comment (dan-gamble) [inline {{cookiecutter.repo_name}}/package.json]: This should just be `prettier-standard --write` The same as the `*.js` one.

**Review by dan-gamble — CHANGES_REQUESTED** [review]

File: {{cookiecutter.repo_name}}/package.json (hunk 1 above)
Comment (lewiscollard) [reply {{cookiecutter.repo_name}}/package.json]: I thought `prettier-standard` was a Javascript-specific formatter.

Comment (dan-gamble) [reply {{cookiecutter.repo_name}}/package.json]: Nah, `prettier-standard` is just `prettier` but with all the rules of StandardJS (Like single quotes)

**Comment by lewiscollard** [comment]
Now has a fix for #179.

**Comment by dan-gamble** [comment]
You could add `git add` to https://github.com/onespacemedia/project-template/blob/4a4c36b23644a9e822aa153772b8afed7adf6c11/%7B%7Bcookiecutter.repo_name%7D%7D/package.json#L115 The Python portion of this. I know that's an outstanding bug

**Comment by dan-gamble** [comment]
Looks okay to me though, could be worth starting a project with it and seeing if you can do `yarn run dev` from the get go

**Comment by lewiscollard** [comment]
@dan-gamble,

> You could add git add to [the] Python portion of this. I know that's an outstanding bug

Done.

> Looks okay to me though, could be worth starting a project with it and seeing if you can do yarn run dev from the get go

Just done so with all of the commits in this PR. `yarn run dev`, `build`, `lint:css`, `lint:js` are all working. :)

**Review by dan-gamble — APPROVED** [review]

**Comment by dan-gamble** [comment]
Awesome, if that's the case i'd say this is g2g 👍

**Pull request merged**